lib/utils/isFirstNested.js

```diff
@@ -4,9 +4,44 @@
 module.exports = function(statement /*: postcss$node*/) /*: boolean*/ {
   const parentNode = statement.parent;
 
-  return (
-    parentNode !== undefined &&
-    parentNode.type !== "root" &&
-    statement === parentNode.first
-  );
+  if (parentNode === undefined || parentNode.type === "root") {
+    return false;
+  }
+
+  if (statement === parentNode.first) {
+    return true;
+  }
+
+  /*
+   * Search for the statement in the parent's nodes, ignoring comment
+   * nodes on the same line as the parent's opening brace.
+   */
+
+  const parentNodes = parentNode.nodes;
+  const firstNode = parentNodes[0];
+
+  if (firstNode.type !== "comment" || firstNode.raws.before.includes("\n")) {
+    return false;
+  }
+
+  const openingBraceLine = firstNode.source.start.line;
+
+  if (openingBraceLine !== firstNode.source.end.line) {
+    return false;
+  }
+
+  for (let i = 1; i < parentNodes.length; i++) {
+    const node = parentNodes[i];
+
+    if (node === statement) {
+      return true;
+    }
+
+    if (node.type !== "comment" || node.source.end.line !== openingBraceLine) {
+      return false;
+    }
+  }
+
+  /* istanbul ignore next: Should always return in the loop */
```

Comment:
It's impossible for the code execution to reach here. One of the `return` statements in the for loop will always be executed. This is guaranteed because it returns if it finds the `statement` node in `parent.nodes` and that must happen because that node must be in its own parent's list of nodes.

If this is too strange I could change the `return false;` in the for loop into `break;` so then this `return false;` at the end of the function will be executed.